### PR TITLE
Add workflow events documentation

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: reference/events/workflow_events.rst [hunk 1]
```diff
@@ -0,0 +1,251 @@
+Workflow events (Enterprise Edition only)
+=========================================
+
+Workflow events are dispatched in the PIM on some workflow actions, mainly in product draft steps (*sent for approval, rejected, deleted, approved...*) and publication feature.
```
Comment (skeleton) [inline reference/events/workflow_events.rst]: send ?

**Comment by ClementGautier** [comment]
GTM

File: reference/events/workflow_events.rst (hunk 1 above)
Comment (grena) [inline reference/events/workflow_events.rst]: I used past for other verbs too, I think it's ok to use sent :)

**Comment by jmleroux** [comment]
GTM

**Pull request merged**